.github/actions/validate-docs-links/README.MD

```diff
@@ -0,0 +1,17 @@
+# GitHub Action for Validating Documentation Links
```

Comment:
I was wondering: what are the advantages of creating our own checker (and maintain it in this repo) compared to the current solutions we could already find on the GitHub action marketplace?
Like this one: https://github.com/marketplace/actions/markdown-link-check
(I took the first link I found, I did not do any investigation)

Reply:
From the ones I checked, they don't match our necessity. 

Taking in consideration how the documentation is structured we're not just checking `mdx` files to one another, we also need to retrieve the slugs from the config JSON files and check those too. 

Also keeping the action "in house" would facilitate us any upgrade needed in the future. (e.g. skip broken links handled by the `redirects.json`)

Reply:
> Taking in consideration how the documentation is structured, we're not just checking mdx files to one another, we also need to retrieve the slugs from the config JSON files and check those too.

I guessed the reason would be something like this, thank you for taking the time to explain 😊